src/tools/info/ucx_info.c

```diff
@@ -66,6 +70,7 @@ int main(int argc, char **argv)
     size_t ucp_num_ppn;
     unsigned print_opts;
     char *tl_name, *mem_size;
+    char ip_addr[MAX_IP_ADDR_LEN] = "\0";
```

Comment:
why need to define a string with two null terminators?

Reply:
const char *ip_addr = NULL;
...
ip_addr = optarg;

Reply:
```
const char *ip_addr = NULL;
...
ip_addr = optarg;
```
not sure that this will work if this parameter will go as not the last one

Reply:
it points int global array of argv

Reply:
i see, will fix